src/main/java/cz/smarteon/loxone/app/Category.java

```diff
@@ -0,0 +1,26 @@
+package cz.smarteon.loxone.app;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import cz.smarteon.loxone.LoxoneUuid;
+
+/**
+ * Represents Category
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Category {
+
+    @JsonProperty(value = "uuid", required = true)
+    protected LoxoneUuid uuid;
```

Comment:
could it be private? There is no class which would benefit from protected visibility. Also rather use constructor binding with `@JsonCreator` (the properties can be final then)